src/Symfony/Component/Intl/Resources/bin/compile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-[[ $1 == force ]] && docker pull jakzal/php-intl
+[[ $1 == force ]] && docker pull jakzal/php-intl:8.2-73.2
```

Comment:
the image with intl 74.1 does not exist (yet) .
But it is not needed to generate stubs.